src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -724,12 +725,24 @@ export class PatientViewPageStore {
             );
         },
     });
+    @observable _selectedMutationalSignatureVersion: string;
+
+    readonly initialMutationalSignatureVersion = remoteData({
+        await: () => [],
+        invoke: () => {
+            return Promise.resolve(
+                retrieveMutationalSignatureVersionFromData(
+                    this.fetchAllMutationalSignatureData.result.map(
+                        profile => profile.molecularProfileId
+                    )
+                )
+            );
+        },
+        default: 'v2',
```

Comment:
we could get rid of this default.   if typescript complains you can use exaimation point at the end to force typescript to ignore the possibility of undefined.  
```
this.initialMutationalSignatureVersion.result!
```